src/workers/transactions/userBoughtCores.ts

```diff
@@ -71,6 +75,11 @@ export const userBoughtCores: TypedWorker<'api.v1.user-transaction'> = {
         },
         to: user.email,
       });
+
+      logger.info(
+        { transactionId: transaction.id },
+        'Email sent for user bought cores',
+      );
```

Comment:
accidenal?

Reply:
no, wanted to log it so that I can monitor production logs if it sends duplicates for same transaction

Reply:
yeah fine for above duplicates, but this one would log each time?

Reply:
The above one could be removed, because it returns early and doesn't send email.
This is after email is sent, for cases where there's a race condition